tests/suite-unit/PluginFlyvemdmTaskstatus.php

```diff
@@ -35,6 +35,40 @@
 
 class PluginFlyvemdmTaskstatus extends CommonTestCase {
 
+    private $dataField = [
+        'group'     => 'application',
+        'symbol'    => 'deployApp',
+        'type_data' => '',
+        'unicity'   => 0,
+     ];
+     /**
+      * @return array
+      */
+     private function createNewTaskInstance() {
+        $class = new \PluginFlyvemdmTask();
+        $task = $this->newTestedInstance($class);
+        return $task;
+     }
+
+     public function beforeTestMethod($method) {
+        switch ($method) {
+            case 'testPrepareInputForUpdate':
+                parent::beforeTestMethod($method);
+                $this->setupGLPIFramework();
+                $this->login('glpi', 'glpi');
+               break;
+         }
+     }
+     public function afterTestMethod($method) {
```

Comment:
This method is useless: tests runs in separate process as long they are parallelized; and before each test the framework is initialized, resetting the session.